oam/install_scripts/mariadb-columnstore-start.sh

```diff
@@ -2,14 +2,21 @@
 
 # This script allows to gracefully start MCS
 
+# prevent nodes using shared storage manager from stepping on each other when initializing
+# flock will open up an exclusive file lock to run atomic operations
+exec {fd_lock}>/var/lib/columnstore/storagemanager/storagemanager-lock
+flock -n "$fd_lock" || exit 0
+
 /bin/systemctl start mcs-workernode
 /bin/systemctl start mcs-controllernode
 /bin/systemctl start mcs-primproc
 /bin/systemctl start mcs-writeengineserver
 /bin/systemctl start mcs-exemgr
 /bin/systemctl start mcs-dmlproc
 /bin/systemctl start mcs-ddlproc
+sleep 2
```

Comment:
I belive we should be fine w/o extra sleeps. What happens w/o it?

Reply:
I inserted here for consistency. When we had startup in columnstore-post-install we had the sleep. I think it was precautionary, to give system time to be ready before dbbuilder is run. i did write the original sleep 2 inside columnstore-post-install so this is just speculation.